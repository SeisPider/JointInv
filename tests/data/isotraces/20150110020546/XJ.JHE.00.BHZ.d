SURF96 R U X -1 94.99999 3.51821 0.20426
SURF96 R U X -1 90.00000 3.66262 0.20972
SURF96 R U X -1 75.00000 3.83343 0.19145
SURF96 R U X -1 70.00000 3.73443 0.16958
SURF96 R U X -1 65.00000 3.78526 0.16178
SURF96 R U X -1 60.00000 3.77790 0.14875
SURF96 R U X -1 55.00000 3.74868 0.13426
SURF96 R U X -1 50.00000 3.69946 0.11887
SURF96 R U X -1 48.00000 3.65769 0.11155
SURF96 R U X -1 46.00000 3.63133 0.10537
SURF96 R U X -1 44.00000 3.60994 0.09960
SURF96 R U X -1 42.00000 3.60102 0.09461
SURF96 R U X -1 40.00000 3.59845 0.08997
SURF96 R U X -1 38.00000 3.57683 0.08445
SURF96 R U X -1 36.00000 3.49063 0.07620
SURF96 R U X -1 34.00000 3.40074 0.06830
SURF96 R U X -1 30.00000 3.37907 0.05950
SURF96 R U X -1 29.00000 3.36641 0.05709
SURF96 R U X -1 28.00000 3.34421 0.05440
SURF96 R U X -1 27.00000 3.32632 0.05189
SURF96 R U X -1 26.00000 3.32158 0.04983
SURF96 R U X -1 25.00000 3.31867 0.04783
