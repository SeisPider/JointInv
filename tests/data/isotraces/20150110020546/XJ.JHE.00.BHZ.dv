MFT96 R U T -1 95.256    3.6915    1.0000    5756.8     0.4567E-06   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 93.091    3.7003    1.0000    5756.8     0.5122E-06   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 91.022    3.7085    1.0000    5756.8     0.5718E-06   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 89.043    3.7162    1.0000    5756.8     0.6365E-06   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 87.149    3.7235    1.0000    5756.8     0.7071E-06   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 85.333    3.7303    1.0000    5756.8     0.7846E-06   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 83.592    3.7365    1.0000    5756.8     0.8699E-06   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 81.920    3.7423    1.0000    5756.8     0.9634E-06   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 80.314    3.7477    1.0000    5756.8     0.1065E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 78.769    3.7525    1.0000    5756.8     0.1176E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 77.283    3.7569    1.0000    5756.8     0.1295E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 75.852    3.7609    1.0000    5756.8     0.1421E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 74.473    3.7643    1.0000    5756.8     0.1554E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 73.143    3.7674    1.0000    5756.8     0.1693E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 71.860    3.7700    1.0000    5756.8     0.1835E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 70.621    3.7721    1.0000    5756.8     0.1980E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 69.424    3.7738    1.0000    5756.8     0.2126E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 68.267    3.7751    1.0000    5756.8     0.2272E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 67.148    3.7760    1.0000    5756.8     0.2416E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 66.065    3.7765    1.0000    5756.8     0.2559E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 65.016    3.7766    1.0000    5756.8     0.2699E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 64.000    3.7763    1.0000    5756.8     0.2836E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 63.015    3.7756    1.0000    5756.8     0.2970E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 62.061    3.7746    1.0000    5756.8     0.3103E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 61.134    3.7732    1.0000    5756.8     0.3235E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 60.235    3.7714    1.0000    5756.8     0.3368E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 59.362    3.7693    1.0000    5756.8     0.3503E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 58.514    3.7669    1.0000    5756.8     0.3641E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 57.690    3.7641    1.0000    5756.8     0.3785E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 56.889    3.7610    1.0000    5756.8     0.3934E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 56.110    3.7577    1.0000    5756.8     0.4091E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 55.351    3.7540    1.0000    5756.8     0.4255E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 54.613    3.7501    1.0000    5756.8     0.4426E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 53.895    3.7459    1.0000    5756.8     0.4603E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 53.195    3.7414    1.0000    5756.8     0.4785E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 52.513    3.7367    1.0000    5756.8     0.4970E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 51.848    3.7317    1.0000    5756.8     0.5156E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 51.200    3.7265    1.0000    5756.8     0.5341E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 50.568    3.7211    1.0000    5756.8     0.5522E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 49.951    3.7155    1.0000    5756.8     0.5697E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 49.349    3.7097    1.0000    5756.8     0.5865E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 48.762    3.7037    1.0000    5756.8     0.6023E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 48.188    3.6975    1.0000    5756.8     0.6171E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 47.628    3.6911    1.0000    5756.8     0.6311E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 47.080    3.6846    1.0000    5756.8     0.6444E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 46.545    3.6780    1.0000    5756.8     0.6571E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 46.022    3.6712    1.0000    5756.8     0.6697E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 45.511    3.6643    1.0000    5756.8     0.6824E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 45.011    3.6572    1.0000    5756.8     0.6957E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 44.522    3.6501    1.0000    5756.8     0.7099E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 44.043    3.6429    1.0000    5756.8     0.7254E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 43.574    3.6355    1.0000    5756.8     0.7423E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 43.116    3.6281    1.0000    5756.8     0.7609E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 42.667    3.6206    1.0000    5756.8     0.7813E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 42.227    3.6131    1.0000    5756.8     0.8034E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 41.796    3.6055    1.0000    5756.8     0.8273E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 41.374    3.5979    1.0000    5756.8     0.8531E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 40.960    3.5902    1.0000    5756.8     0.8806E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 40.554    3.5825    1.0000    5756.8     0.9099E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 40.157    3.5748    1.0000    5756.8     0.9411E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 39.767    3.5670    1.0000    5756.8     0.9740E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 39.385    3.5593    1.0000    5756.8     0.1009E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 39.010    3.5516    1.0000    5756.8     0.1045E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 38.642    3.5439    1.0000    5756.8     0.1082E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 38.280    3.5362    1.0000    5756.8     0.1121E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 37.926    3.5285    1.0000    5756.8     0.1159E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 37.578    3.5209    1.0000    5756.8     0.1197E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 37.236    3.5133    1.0000    5756.8     0.1233E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 36.901    3.5058    1.0000    5756.8     0.1267E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 36.571    3.4983    1.0000    5756.8     0.1298E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 36.248    3.4909    1.0000    5756.8     0.1325E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 35.930    3.4836    1.0000    5756.8     0.1349E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 35.617    3.4763    1.0000    5756.8     0.1368E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 35.310    3.4691    1.0000    5756.8     0.1383E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 35.009    3.4620    1.0000    5756.8     0.1396E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 34.712    3.4550    1.0000    5756.8     0.1406E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 34.420    3.4482    1.0000    5756.8     0.1416E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 34.133    3.4414    1.0000    5756.8     0.1428E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 33.851    3.4347    1.0000    5756.8     0.1443E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 33.574    3.4282    1.0000    5756.8     0.1464E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 33.301    3.4218    1.0000    5756.8     0.1491E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 33.032    3.4155    1.0000    5756.8     0.1525E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 32.768    3.4094    1.0000    5756.8     0.1567E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 32.508    3.4034    1.0000    5756.8     0.1617E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 32.252    3.3975    1.0000    5756.8     0.1673E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 32.000    3.3918    1.0000    5756.8     0.1734E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 31.752    3.3863    1.0000    5756.8     0.1798E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 31.508    3.3810    1.0000    5756.8     0.1864E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 31.267    3.3758    1.0000    5756.8     0.1931E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 31.030    3.3708    1.0000    5756.8     0.1999E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 30.797    3.3659    1.0000    5756.8     0.2065E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 30.567    3.3613    1.0000    5756.8     0.2131E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 30.341    3.3569    1.0000    5756.8     0.2197E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 30.118    3.3526    1.0000    5756.8     0.2262E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 29.898    3.3486    1.0000    5756.8     0.2327E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 29.681    3.3448    1.0000    5756.8     0.2391E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 29.468    3.3412    1.0000    5756.8     0.2454E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 29.257    3.3378    1.0000    5756.8     0.2516E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 29.050    3.3346    1.0000    5756.8     0.2574E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 28.845    3.3317    1.0000    5756.8     0.2628E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 28.643    3.3290    1.0000    5756.8     0.2675E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 28.444    3.3265    1.0000    5756.8     0.2714E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 28.248    3.3243    1.0000    5756.8     0.2743E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 28.055    3.3224    1.0000    5756.8     0.2760E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 27.864    3.3207    1.0000    5756.8     0.2764E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 27.676    3.3193    1.0000    5756.8     0.2753E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 27.490    3.3181    1.0000    5756.8     0.2728E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 27.307    3.3172    1.0000    5756.8     0.2688E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 27.126    3.3167    1.0000    5756.8     0.2633E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 26.947    3.3164    1.0000    5756.8     0.2566E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 26.771    3.3164    1.0000    5756.8     0.2485E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 26.597    3.3167    1.0000    5756.8     0.2394E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 26.426    3.3173    1.0000    5756.8     0.2293E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 26.256    3.3182    1.0000    5756.8     0.2183E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 26.089    3.3195    1.0000    5756.8     0.2067E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 25.924    3.3211    1.0000    5756.8     0.1946E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 25.761    3.3230    1.0000    5756.8     0.1821E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 25.600    3.3253    1.0000    5756.8     0.1693E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 25.441    3.3280    1.0000    5756.8     0.1563E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 25.284    3.3310    1.0000    5756.8     0.1432E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 25.129    3.3343    1.0000    5756.8     0.1301E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 95.000    3.6926    1.0000    5756.8     0.4632E-06   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 90.000    3.7125    1.0000    5756.8     0.6052E-06   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 85.000    3.7315    1.0000    5756.8     0.8009E-06   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 80.000    3.7487    1.0000    5756.8     0.1088E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 75.000    3.7630    1.0000    5756.8     0.1504E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 70.000    3.7730    1.0000    5756.8     0.2056E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 65.000    3.7766    1.0000    5756.8     0.2701E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 60.000    3.7708    1.0000    5756.8     0.3404E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 55.000    3.7521    1.0000    5756.8     0.4336E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 50.000    3.7159    1.0000    5756.8     0.5684E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 48.000    3.6954    1.0000    5756.8     0.6218E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 46.000    3.6709    1.0000    5756.8     0.6702E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 44.000    3.6422    1.0000    5756.8     0.7270E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 42.000    3.6091    1.0000    5756.8     0.8160E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 40.000    3.5717    1.0000    5756.8     0.9543E-05   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 38.000    3.5301    1.0000    5756.8     0.1151E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 36.000    3.4852    1.0000    5756.8     0.1344E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 34.000    3.4382    1.0000    5756.8     0.1435E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 32.000    3.3918    1.0000    5756.8     0.1734E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 30.000    3.3505    1.0000    5756.8     0.2297E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 29.000    3.3339    1.0000    5756.8     0.2587E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 28.000    3.3219    1.0000    5756.8     0.2761E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 27.000    3.3165    1.0000    5756.8     0.2586E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 26.000    3.3204    1.0000    5756.8     0.2002E-04   -5.655000   68.361298   44.619999   82.970001 0  1 COMMENT
