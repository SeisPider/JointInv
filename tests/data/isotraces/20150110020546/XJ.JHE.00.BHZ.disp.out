SURF96 R U T -1 95.000    3.6926    1.0000    
SURF96 R C T -1 95.000    5.9565    1.0000    
SURF96 R U T -1 90.000    3.7125    1.0000    
SURF96 R C T -1 90.000    5.7718    1.0000    
SURF96 R U T -1 85.000    3.7315    1.0000    
SURF96 R C T -1 85.000    5.6003    1.0000    
SURF96 R U T -1 80.000    3.7487    1.0000    
SURF96 R C T -1 80.000    5.4412    1.0000    
SURF96 R U T -1 75.000    3.7630    1.0000    
SURF96 R C T -1 75.000    5.2928    1.0000    
SURF96 R U T -1 70.000    3.7730    1.0000    
SURF96 R C T -1 70.000    5.1538    1.0000    
SURF96 R U T -1 65.000    3.7766    1.0000    
SURF96 R C T -1 65.000    5.0228    1.0000    
SURF96 R U T -1 60.000    3.7708    1.0000    
SURF96 R C T -1 60.000    4.8982    1.0000    
SURF96 R U T -1 55.000    3.7521    1.0000    
SURF96 R C T -1 55.000    4.7780    1.0000    
SURF96 R U T -1 50.000    3.7159    1.0000    
SURF96 R C T -1 50.000    4.6597    1.0000    
SURF96 R U T -1 48.000    3.6954    1.0000    
SURF96 R C T -1 48.000    4.6122    1.0000    
SURF96 R U T -1 46.000    3.6709    1.0000    
SURF96 R C T -1 46.000    4.5643    1.0000    
SURF96 R U T -1 44.000    3.6422    1.0000    
SURF96 R C T -1 44.000    4.5155    1.0000    
SURF96 R U T -1 42.000    3.6091    1.0000    
SURF96 R C T -1 42.000    4.4657    1.0000    
SURF96 R U T -1 40.000    3.5717    1.0000    
SURF96 R C T -1 40.000    4.4145    1.0000    
SURF96 R U T -1 38.000    3.5301    1.0000    
SURF96 R C T -1 38.000    4.3614    1.0000    
SURF96 R U T -1 36.000    3.4852    1.0000    
SURF96 R C T -1 36.000    4.3063    1.0000    
SURF96 R U T -1 34.000    3.4382    1.0000    
SURF96 R C T -1 34.000    4.2486    1.0000    
SURF96 R U T -1 32.000    3.3918    1.0000    
SURF96 R C T -1 32.000    4.1884    1.0000    
SURF96 R U T -1 30.000    3.3505    1.0000    
SURF96 R C T -1 30.000    4.1258    1.0000    
SURF96 R U T -1 29.000    3.3339    1.0000    
SURF96 R C T -1 29.000    4.0938    1.0000    
SURF96 R U T -1 28.000    3.3219    1.0000    
SURF96 R C T -1 28.000    4.0615    1.0000    
SURF96 R U T -1 27.000    3.3165    1.0000    
SURF96 R C T -1 27.000    4.0293    1.0000    
SURF96 R U T -1 26.000    3.3204    1.0000    
SURF96 R C T -1 26.000    3.9975    1.0000    
